leaflet/static/leaflet/leaflet.css

```diff
@@ -637,4 +637,12 @@
 /* Resize the "display_raw" textbox */
 .django-leaflet-raw-textarea {
 	width: 100%;
+}
+
+.leaflet-editing-icon {

```

Comment:
What does this have to do in this PR?

Reply:
I used it for debugging process.
Has been removed.

Reply:
You remove `.django-leaflet-raw-textarea` and let `.leaflet-editing-icon`.

Reply:
Yes, it changes the shape of editing markets from a rectangle to a circle, which is more beautiful and elegant in my own opinion, do you want me to remove it?

Reply:
Yes, it changes the shape of editing markets from a rectangle to a circle, which is more beautiful and elegant in my own opinion, do you want me to remove it?

Reply:
Yes please, interface customization should be done in your project.